vm-setup/roles/common/defaults/main.yml

```diff
@@ -26,6 +26,8 @@ ironic_prefix: ""
 
 baremetal_network_cidr_v4: "{{ lookup('env', 'EXTERNAL_SUBNET_V4')|default('', true) }}"
 baremetal_network_cidr_v6: "{{ lookup('env', 'EXTERNAL_SUBNET_V6')|default('', true) }}"
+virsh_dhcp_v4_start: "{{ lookup('env', 'VIRSH_DHCP_V4_START')|default('192.168.111.20/24', true) }}"
+virsh_dhcp_v4_end: "{{ lookup('env', 'VIRSH_DHCP_V4_END')|default('192.168.111.60/24', true) }}"
```

Comment:
the default is inconsistent with the default of baremetal_network_cidr_v4

Reply:
I didn't get it

Reply:
Ah ok, you dont want to put any default here as is the case of baremetal_network_cidr_v4, go it

Reply:
Fixed